debian/control

```diff
@@ -6,22 +5,30 @@ Maintainer: Go Compiler Team <pkg-golang-devel@lists.alioth.debian.org>
 Uploaders: Michael Stapelberg <stapelberg@debian.org>,
            Paul Tagliamonte <paultag@debian.org>,
            Tianon Gravi <admwiggin@gmail.com>
-Vcs-Browser: http://anonscm.debian.org/gitweb/?p=pkg-golang/golang.git
+Vcs-Browser: https://anonscm.debian.org/cgit/pkg-golang/golang.git
 Vcs-Git: git://anonscm.debian.org/pkg-golang/golang.git
-Build-Depends: debhelper (>= 7.4.10), bison, ed, mawk | awk, perl, netbase
+Build-Depends: bison,
+               debhelper (>= 7.4.10),
+               ed,
+               golang (>= 1:1.4~) | gccgo-5,
+               mawk | awk,
+               netbase,
+               perl
 Build-Depends-Indep: po-debconf
 Standards-Version: 3.9.6
-Homepage: http://golang.org/
+Homepage: https://golang.org
 
 Package: golang-go
-Architecture: i386 amd64 armel armhf
-Depends: ${shlibs:Depends}, ${misc:Depends}, ${perl:Depends},
-	 golang-src (>= ${source:Version}),
-	 golang-go-linux-${go:Hostarch} (= ${source:Version})
-Replaces: golang-weekly-go, golang-weekly-tools, golang-tools
-Conflicts: golang-weekly-go, golang-weekly-tools, golang-tools
-Provides: go-compiler, golang-weekly-go, golang-weekly-tools, golang-tools
-Suggests: bzr, git, golang-go.tools, mercurial, subversion
+Architecture: amd64 arm64 armel armhf i386 ppc64 ppc64el
+Depends: golang-pkg (= ${source:Version}),
+         golang-src (>= ${source:Version}),
```

Comment:
Having all of these as separate packages feels really strange to me, especially now that cross compile doesn't require the `golang-go-HOST-ARCH` packages that install pre-compiled files into `GOROOT` -- think it's worth considering combining them now?

(For context, the new `golang-pkg` binary package is my replacement for `golang-go-HOST-ARCH`, since we only need one compiled for the host arch.)